commands/build.go

```diff
@@ -155,8 +156,12 @@ func DockerBuild(jobDirectory, version, username, password, manifest, dockerfile
 	}
 
 	util.PrintUtil("INFO: Successfully built image. This image can be published with the following command:\n")
-	util.PrintUtil("seed publish -in %s -r hub.docker.com -o geoint\n", imageName)
+	util.PrintUtil("seed publish -in %s -r hub.docker.com\n", imageName)
 	util.PrintUtil("This image can be run with the following command:\n")
+	runCmd := fmt.Sprintf("seed run -rm -in %s %s %s %s-o <outdir>", imageName, inputStr, settingStr, mountStr)
+	space := regexp.MustCompile(`\s+`)
```

Comment:
This is a very useful sanitizing snippet. Could we pull that into`util`? I imaging there are other places we could use that.